### PR TITLE
Fix slipping through fences

### LINKED ISSUE #24837: you can slip over chainlink fences
## Description
title

### PR DESCRIPTION
**Pull request by Arendian**
## About the PR

resolves #24837 
You can't slip through chain link fences anymore, but in exchange flying mobs can't pass them anymore either.

## Why / Balance

We don't turn into a terminator the moment we fall down.

## Technical details

CollisionLayer changed from TableLayer to HighImpassable and MidImpassable.

## Media

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: Dygon
- fix: You can't slip through chain link fences anymore.

### CODE REVIEW

File: Resources/Prototypes/Entities/Structures/Walls/fence_metal.yml [hunk 1]
```diff
@@ -163,7 +163,8 @@
         mask:
         - FullTileMask
         layer:
-        - TableLayer
+        - HighImpassable
+        - MidImpassable
   - type: InteractionPopup
     interactSuccessString: fence-rattle-success
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Structures/Walls/fence_metal.yml:169]: Does LowImpassable work instead?

Comment (Arendian) [reply Resources/Prototypes/Entities/Structures/Walls/fence_metal.yml:169]: Opaque makes it so both flying and small mobs can pass, but then you can't shoot through it with a laser

Comment (metalgearsloth) [reply Resources/Prototypes/Entities/Structures/Walls/fence_metal.yml:169]: Considering it's a fence I think it's better to block small mobs than flying.

Comment (Arendian) [reply Resources/Prototypes/Entities/Structures/Walls/fence_metal.yml]: I'm fine with that too, it's either a fence with holes big enough for mice to pass that reaches the ceiling, or a chain fence that doesn't reach the ceiling with extremely tiny "chains". Let's do the latter.

**Pull request merged**